.github/workflows/build.yml

```diff
@@ -71,19 +71,29 @@ jobs:
         name: Checkout
         uses: actions/checkout@v2
         with:
-          ref: ${{ github.event.base_ref }}
+          fetch-depth: 0
+      -
+        name: Prepare
+        id: prepare
+        run: |
+          echo ::set-output name=tag_name::${GITHUB_REF#refs/tags/}
+          echo ::set-output name=branch_name::$(git --no-pager branch -a --contains "$GITHUB_SHA" | grep 'remotes/origin/' | cut -d '/' -f3)
```

Comment:
The `cut` section is going to incorrectly truncate the branch name if it contains a slash. If I use the branch `release/1_3` as an example, this would find the commit on that branch, but then truncate it to `release` which wont match any available branch